frappe/core/doctype/doctype/doctype.py

```diff
@@ -784,10 +784,9 @@ def validate_links_table_fieldnames(meta):
 	if not meta.links or frappe.flags.in_patch or frappe.flags.in_fixtures:
 		return
 
-	fieldnames = tuple(field.fieldname for field in meta.fields)
 	for index, link in enumerate(meta.links, 1):
 		link_meta = frappe.get_meta(link.link_doctype)
-		if not link_meta.get_field(link.link_fieldname):
+		if not frappe.get_meta(link.link_doctype).has_field(link.link_fieldname):
```

Comment:
Why? That's basically `link_meta` in the line above.

Reply:
Yeah, wanted to remove that... missed it